ida/data.yml

```diff
@@ -318,6 +316,11 @@ functions:
   0x14192E720: CollisionFree
 
 classes:
+  std::vector:
```

Comment:
It might not be a good idea to have something called `std::vector` when there exist many variations on `<T>` part.